Sample Data/githubscanaudit_CL.json

```diff
@@ -0,0 +1,171 @@
+
+  [
```

Comment:
As mentioned before, can we include all 20 samples.

Reply:
done